tests/test_metadata.py

```diff
@@ -188,6 +188,29 @@ def test_builder(self):
                 for primary_key in primary_keys:
                     self.assertTrue(md_table.columns[primary_key].primary_key)
 
+    def test_sized_types(self):
```

Comment:
I like this addition of a test for looking at sized types. Let's include this even if it doesn't not get incorporated via this PR. It can be moved to a user branch instead.

Reply:
I think felis need a lot more unit tests, current test coverage is severely lacking.